source/options.rst

```diff
@@ -84,6 +93,81 @@ To view the created configmap, use the following command:
 
    kubectl describe configmaps cluster1-pxc
 
+Use a Secret Object
+-------------------
+
+The Operator can also store configuration options in `Kubernetes Secrets <https://kubernetes.io/docs/concepts/configuration/secret/>`_.
+This can be useful if you need additional protection for some sensitive data.
+
+You should create a Secret object with a specific name, composed of your cluster
+name and a specific suffix:
+
+* ``my-cluster-name-pxc`` for the Percona XtraDB Cluster Pods,
+* ``my-cluster-name-proxysql`` for the ProxySQL Pods,
```

Comment:
We have the following docs:
1. https://www.percona.com/doc/kubernetes-operator-for-pxc/haproxy-conf.html#passing-custom-configuration-options-to-haproxy
2. https://www.percona.com/doc/kubernetes-operator-for-pxc/proxysql-conf.html#passing-custom-configuration-options-to-proxysql

There are two options:
1. We add corresponding information about configmap and secret configuration to the docs above. Separately for ProxySQL and Haproxy.
2. We rename the doc "Changing MySQL Options" to smth like "Customizing Operator Components" and tell the story about config maps and secrets only in this doc.

What do you think?